DetectorDescription/Detectors/src/SimpleCone_geo.cxx

```diff
@@ -0,0 +1,51 @@
+// $Id: $
+//====================================================================
+//  AIDA Detector description implementation for FCC
+//-------------------------------------------------------------------- 
+// Simple cone using dimensions to be used to define cone composed of 1 single material
+//
+// @author Clement Helsens
+//====================================================================
+#include "DD4hep/DetFactoryHelper.h"
+
+using namespace std;
+using namespace DD4hep;
+using namespace DD4hep::Geometry;
+
+
+static Ref_t create_element(LCDD& lcdd, xml_h e, SensitiveDetector sens)  {
```

Comment:
Can you give this a more specific name and follow the naming conventions? E.g. `createSimpleCone`.

Also the function should be in a namespace, e.g. `det`.